package com.github.zeroberto.productsstore.exceptions;

public final class DataStoreNetworkException extends RuntimeException {

  public DataStoreNetworkException(Throwable cause) {
    super(cause);
  }
}
